opentreemap/treemap/lib/user.py

```diff
@@ -21,12 +19,14 @@ def _instance_ids_edited_by(user):
                         .distinct()
 
 
-def get_audits(logged_in_user, instance, query_vars, user, models,
-               model_id, page=0, page_size=20, exclude_pending=True,
-               should_count=False):
-    start_pos = page * page_size
-    end_pos = start_pos + page_size
+PAGE_DEFAULT = 20
+ALLOWED_MODELS = get_filterable_audit_models()
+
 
+def get_audits(logged_in_user, instance, query_vars, user=None,
```

Comment:
Nice catch! Fixed in https://github.com/OpenTreeMap/otm-core/pull/2904/commits/05859cc71145e7f874d79c2ea6d80b117b75e922 (with unit tests!)